tests/jax/test_custom_ops_cpar.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
```

Comment:
Add a comment to tell what cpar mean?

Reply:
Will do. Related to this, I am not attached to any file, class, method, etc. names here. Please let me know if you have any better, more concise and/or descriptive alternatives, especially if there's naming conventions in TE-JAX that I've missed or forgotten about.